content/general/concepts/mfa/mfa.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'MFA'
+Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
```

Comment:
```suggestion
  - 'Computer Science'
  - 'Cybersecurity'
```

Wouldn't this concept more closely related to the field of cybersecurity as opposed to web development?